imap/http_caldav.c

```diff
@@ -2270,7 +2270,7 @@ static int list_calendars(struct transaction_t *txn)
     buf_printf(&txn->buf, "%s://%s%s", proto, host, txn->req_tgt.path);
 
     memset(&lrock, 0, sizeof(struct list_cal_rock));
-    lrock.scheddefault = caldav_scheddefault(httpd_userid);
+    lrock.scheddefault = caldav_scheddefault(httpd_userid, 0);
```

Comment:
Why is this `0` and not `1`?  My understanding is that even if the user does not set a “Default Calendar Collection”, Cyrus IMAP does provide one and that one cannot be deleted and is highlighted.

Reply:
I agree with you, this is a regression. Will you provide a pull request?

Reply:
> Will you provide a pull request?

No.

Reply:
How about here: https://github.com/cyrusimap/cyrus-imapd/blob/master/imap/http_caldav.c#L1059 ?